app/lib/taric_importer.rb

```diff
@@ -58,19 +59,24 @@ def taric_failed_log(exception, hash)
   private
 
   def proceed_with_import?(filename)
```

Comment:
This check always returns  true (because use_cds? is false) 

So the method can go